src/hooks/sdk/useMediaStreamMetrics.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
So you create a new one with bandwidth metrics, do we want to deprecate useBandwidthMetrics then? Otherwise, how do one know which one to use?

Reply:
yes, I would prefer to deprecate the useBandwidthMetrics one.  What is the best practice I should follow to not making it a breaking change?